test/unit/instrumentation/core/promises.test.js

```diff
@@ -235,41 +156,14 @@ test('Promise trace', (t) => {
         })
         .then(step('c'))
         .then(step('d'))
-        .then(checkTrace(t, tx, expected))
+        .then(checkTrace(t, tx))
         .then(() => {
           t.end()
         })
     })
   })
 
   t.test('should handle continuing from returned promises', (t) => {
-    //   (return)
-    //       /--e---f---g--\
-    //      /               \
-    // a---b- - - - - - - - -c---d
```

Comment:
Part of me is wondering if keeping the graph part of these would still be useful in explaining the chaining that is happening but perhaps not.

Reply:
i don't know. the chaining just continues to propagate the original segment